topics/imaging/tutorials/detection-of-mitoflashes/tutorial.md

```diff
@@ -0,0 +1,182 @@
+
+By linking each mitochondrion’s position across frames, we ensure that intensity fluctuations represent true changes within the same mitochondrial structure, enabling precise analysis of events like mitoflashes.
+
+> <hands-on-title>Tracking Mitochondrial Movement</hands-on-title>
+>
+> 1. {% tool [Perform linking in time series (nearest neighbors)](toolshed.g2.bx.psu.edu/repos/imgteam/points_association_nn/ip_points_association_nn/0.0.3-2) %} with the following recommended parameters:
+>    - {% icon param-file %} *"Coordinates (and intensities) of input points"*: Output from the **Spot Detection** tool.
+>    - **Neighborhood size**: Set to `6` pixels as a starting value. Adjust within `1-10` pixels based on the density of points and proximity between spots across frames.
+>    - **Intensity threshold**: A recommended starting value is `25%` of the global maximum intensity. Increase to `30-40%` for stricter filtering if spots have low intensity.
+>    - **Minimum track length**: Use `50%` of the sequence length as a baseline. Adjust based on the expected duration of mitoflash events.
+>    
+>    This tool links detected spots across frames to track movement. The **Neighborhood size** parameter controls the pixel range for associating points between frames. Reducing the **Intensity threshold** filters out weaker signals, reducing the number of spurious tracks.
+>
+{: .hands_on}
+
+# Detecting Mitoflashes
+
+Mitoflashes are identified based on sudden changes in fluorescence intensity in mitochondria, signifying superoxide bursts. We will use a curve-fitting tool to capture these fluctuations and measure their characteristics.
+
+## Step 3: Curve Fitting for Mitoflash Detection
+
+> <hands-on-title>Mitoflash Detection</hands-on-title>
+>
+> 1. {% tool [Perform curve fitting](toolshed.g2.bx.psu.edu/repos/imgteam/curve_fitting/ip_curve_fitting/0.0.3-2) %} with the following parameters:
+>    - {% icon param-file %} *"File name of input data points (xlsx)"*: Output from the **Perform linking in time series (nearest neighbors)** tool.
+>    - **Degree of the polynomial function**: Set to `2nd degree` to capture intensity peaks that characterize mitoflash events.
+>    - **Penalty**: Choose *Least absolute deviations (LAD)* for robust fitting to intensity fluctuations.
+>    - **Alpha**: Set a significance level, such as `0.01`, to generate assistive curves if needed.
+>
+>    This step applies polynomial curve fitting to model intensity fluctuations over time, identifying mitoflash events when intensity values exceed the fitted curve by the given significance level. Key parameters to analyze are **Amplitude** (peak intensity), **Tpk** (time to peak), and **T50** (duration). These values provide initial insights into mitochondrial function, with further interpretation outlined in the following section.
```

Comment:
The important text part from the paper that explains the curves:
> To better distinguish mitoflashes, we use a second-degree polynomial curve […] to fit the intensities of one mitochondrion over time […] For curve fitting, we [use] the least absolute deviations (LAD) which is more robust to noise and outliers. **This means that the fitted curves will be much less affected by mitoflashes and well represent the basic intensity change patterns**, making the detection of mitoflashes easier and more robust.

And for the assistive curves:
> To objectively quantify significant deviations, for each image sequence we compute the distributions of the measured mitochondria intensities deviating from the fitted curves and **determine a threshold that corresponds to a significance level of 0.01. An intensity increase above this threshold is regarded as "significant," which is used as a base criterion for distinguishing mitoflash events.**

Based on this information, I suggest the following changes:
```suggestion
>    This step applies polynomial curve fitting to model intensity fluctuations over time. The chosen **Penalty** used for curve fitting ensures that the curve is not affected by strong but short intensity deviations (e.g., mitoflashes or image noise) but rather represents the basic underlying intensity patterns. Mitoflash events can then be found by looking for intensity values which exceed the fitted curve by the given significance level. To facilitate this by thresholding, an assistive curve is generated which that corresponds to the significance level. Intensity values above the thresholds given by the assistive curve exceed the given significance level and correspond to mitoflash events.
```